plugins/woocommerce/client/admin/client/task-lists/task-lists.scss

```diff
@@ -254,3 +255,9 @@
 		}
 	}
 }
+
+@include breakpoint( "<782px" ) {
```

Comment:
I'm assuming this area in general is styled desktop first and this just follows that pattern? I ask since the marketplace itself tends to favour mobile first so just checking to be sure. Not a blocker.